### PR TITLE
Fixes: #171 Init Withdraw Reminders

### PR DESCRIPTION
**Pull request by pingustar**
#171

### CODE REVIEW

File: src/elements/earn/portfolio/v3/initWithdraw/step4/V3WithdrawStep4.tsx [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+const generateCalendarEvent = (
+  type: 'ical' | 'google' | 'outlook',
+  days: number
+) => {
+  const start = dayjs().add(days, 'day').toDate();
+  const end = dayjs(start).add(15, 'minute').toDate();
+
+  const event: CalendarOptions = {
+    title: 'Bancor Withdrawal',
+    description: 'Bancor Withdrawal cooldown ended.',
```
Comment (zavelevsky) [inline src/elements/earn/portfolio/v3/initWithdraw/step4/V3WithdrawStep4.tsx:25]: Inconsistent casing in the description. Should probably be withdrawal with lowercase w

Comment (pingustar) [reply src/elements/earn/portfolio/v3/initWithdraw/step4/V3WithdrawStep4.tsx]: fixed

**Review by zavelevsky — APPROVED** [review]
LGTM - didn't test

**Comment by pingustar** [comment]
- [ ] hide icalendar from mobile

**Pull request merged**